ui/components/AlertsTable.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
+import styled from "styled-components";
+import { useFeatureFlags } from "../hooks/featureflags";
+import { Alert, CrossNamespaceObjectRef } from "../lib/objects";
+import { statusSortHelper } from "../lib/utils";
+import DataTable, {
+  Field,
+  filterByStatusCallback,
+  filterConfig,
+} from "./DataTable";
+import KubeStatusIndicator from "./KubeStatusIndicator";
+type Props = {
+  className?: string;
+  rows?: Alert[];
+};
+
+function AlertsTable({ className, rows = [] }: Props) {
+  const { data: flagData } = useFeatureFlags();
+  const flags = flagData?.flags || {};
+  let initialFilterState = {
+    ...filterConfig(rows, "name"),
+    ...filterConfig(rows, "namespace"),
+    ...filterConfig(rows, "eventSeverity"),
+    ...filterConfig(rows, "status", filterByStatusCallback),
+  };
+  if (flags.WEAVE_GITOPS_FEATURE_CLUSTER === "true") {
+    initialFilterState = {
+      ...initialFilterState,
+      ...filterConfig(rows, "clusterName"),
+    };
+  }
+
+  const alertFields: Field[] = [
+    {
+      label: "Name",
+      value: "name",
+    },
+    {
+      label: "Namespace",
+      value: "namespace",
+    },
+    {
+      label: "Severity",
+      value: "severity",
+    },
+    {
+      label: "Event Sources",
+      value: (a) => {
+        return (
+          <ul className="event-sources">
+            {a?.eventSources?.map((obj: CrossNamespaceObjectRef) => (
+              <li className="event-sources" key={obj.name}>
+                {obj.kind}: {obj.name}
```

Comment:
Should this not (also, I wrote more in the other comment) have a namespace?

Reply:
This can definitely have namespace - how do you feel about:
`{obj.kind}: {obj.name}, {obj.namespace}`
Was it supposed to be namespace instead of name? Trying to follow the original design:
<img width="195" alt="image" src="https://user-images.githubusercontent.com/65822698/189738966-c566ea34-c14c-47cf-8488-350466741e41.png">

Reply:
I think the comma could be confusing - looks a bit like it's doing some kind of `{kind}: {objs.filter(o => o.kind == kind).map(o => o.name).join(', ')}`.

I think I would do `{obj.kind}: {obj.namespace}/{obj.name}`

I almost said "because that's how it's usually displayed" and then I realised that it's not (if you run `flux get all -A` you get the `{obj.kind}/{obj.name}` in one column, and namespace in another one, and if you run `kubectl port-forward --help` then you'll see that you can also use `{obj.kind}/{obj.name}`) - but I can't think of anything better that's still somewhat faithful to the original design.